Segmented_Sieve/README.md

```diff
@@ -0,0 +1,58 @@
+# Segmented Sieve
+
+In the classical sieve of Eratosthenes one starts with the numbers 2, 3, 4, ..., N, and successively discard those that are multiples of the primes up to sqrt(N); these primes will be called sieving primes. On a computer, the character of each integer (either prime or composite), or better yet, the character of each integer that is not a multiple of the very first primes (say, 2, 3 and 5), can be represented by one bit of information. Because each computer has a finite amount of physical memory, if N is too large one is forced to subdivide the interval [2,N] in manageable subintervals. This subdivision gives rise to the so-called segmented sieve of Eratosthenes.
+
+
+
+## Algorithm
+To find all the prime numbers in a given interval by Segmented Sieve method:
+- Use Simple Sieve to find all primes upto square root of ‘n’ and store these primes in an array “prime[]”. Store the found primes in an array ‘prime[]’.
+- We need all primes in range [0..n-1]. We divide this range in different segments such that size of every segment is at-most √n
+- Do following for every segment [low..high]
+    - Create an array mark[high-low+1]. Here we need only O(x) space where x is number of elements in given range.
+    - Iterate through all primes found in step 1. For every prime, mark its multiples in given range [low..high].

```

Comment:
Hi, please follow indentation in line 12 & 13 and then it is good to merge.

Reply:
Hi, @MastersAbh  ! Actually I have followed the indentation but lines 12 and 13 are the sub bullet points of lines 11. So, these two points are under that line 11.

Reply:
Oh Sure !